source/run/include/TG4RunManager.h

```diff
@@ -43,8 +44,8 @@ class TApplication;
 class TG4RunManager : public TG4Verbose
 {
  public:
-  TG4RunManager(
-    TG4RunConfiguration* configuration, int argc = 0, char** argv = 0);
+  TG4RunManager(TGeant4* geant4vmc, TG4RunConfiguration* configuration,
```

Comment:
This should not be needed; the class can access the MC data/functions via gMC pointer.